lib/ruby_lsp/requests/request.rb

```diff
@@ -18,19 +18,19 @@ def perform; end
       private
 
       # Checks if a location covers a position
-      sig { params(location: Prism::Location, position: T.untyped).returns(T::Boolean) }
-      def cover?(location, position)
+      sig { params(location: Prism::Location, position: T.untyped, encoding: Encoding).returns(T::Boolean) }
+      def cover?(location, position, encoding)
```

Comment:
Could we pass the encoding to the `Request` initializer, and save it an instance variable, so that don't need to pass it around in methods so much?

Reply:
@andyw8 Since the position is now pre-calculated using byte offsets, there is no longer a need to pass the encoding to each method.

ref: https://github.com/Shopify/ruby-lsp/pull/2051/commits/be648a6ea20bb6a9c4fa31302a5e400a011e022b